examples/calendar_set_date.html

```diff
@@ -0,0 +1,116 @@
+<html>
+
+	<head>
+		<title>Dynamic Organizer Example</title>
+		<link href="../calendarorganizer.min.css" rel="stylesheet">
+	</head>
+
+	<body>
+		<div id="calendarContainer"></div>
+		<div id="organizerContainer"></div>
+		<script src="../calendarorganizer.js"></script>
+		<script>
+			"use strict";
+
+			// function that creates dummy data for demonstration
+			function createDummyData() {
+				var date = new Date();
+				var data = {};
+
+				for (var i = 0; i < 10; i++) {
+					data[date.getFullYear() + i] = {};
+
+					for (var j = 0; j < 12; j++) {
+						data[date.getFullYear() + i][j + 1] = {};
+
+						for (var k = 0; k < Math.ceil(Math.random() * 10); k++) {
+							var l = Math.ceil(Math.random() * 28);
+
+							try {
+								data[date.getFullYear() + i][j + 1][l].push({
+									startTime: "10:00",
+									endTime: "12:00",
+									text: "Some Event Here"
+								});
+							} catch (e) {
+								data[date.getFullYear() + i][j + 1][l] = [];
+								data[date.getFullYear() + i][j + 1][l].push({
+									startTime: "10:00",
+									endTime: "12:00",
+									text: "Some Event Here"
+								});
+							}
+						}
+					}
+				}
+
+				return data;
+			}
+
+			// INSTEAD OF GRABBING THE DATA FROM AN AJAX REQUEST
+			// I WILL BE DEMONSTRATING THE SAME EFFECT THROUGH MEMORY
+			// THIS DEFEATS THE PURPOSE BUT IS SIMPLER TO UNDERSTAND
+			var serverData = createDummyData();
+			var test_day = new Date(Date.parse("2020/11/21"));
+
+			// stating variables in order for them to be global
+			var calendar, organizer;
+
+			// initializing a new calendar object, that will use an html container to create itself
+			calendar = new Calendar("calendarContainer", // id of html container for calendar
+				"small", // size of calendar, can be small | medium | large
+				[
+					"Wednesday", // left most day of calendar labels
+					3 // maximum length of the calendar labels
+				],
+				[
+					"#ffc107", // primary color
+					"#ffa000", // primary dark color
+					"#ffffff", // text color
+					"#ffecb3" // text dark color
+				],
+				{	
+					indicator: true,
+					indicator_type: 0, // indicator type, can be 0 (not numeric) | 1 (numeric)
+					indicator_pos: "top", // indicator position, can be top | bottom
+					custom_date: test_day
```

Comment:
Can you please change the `custom_date` to something like `initial_date` or `pre_selected_date`, those are a bit more informative than `custom_date` 👍🏼